frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -662,6 +676,12 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
         },
     })),
     selectors(({ cache }) => ({
+        webVitalsEvents: [
+            (s) => [s.sessionEventsData],
+            (sessionEventsData): RecordingEventType[] =>
+                (sessionEventsData || []).filter((e) => e.event === '$web_vitals'),
```

Comment:
I'd love to start using constants for such strings, we have so many across our code that replacing or making typos is super easy.
says the person used to strongly typed languages :P

Reply:
tbh we could have a type in the front end that expresses all the known event names + and string so you don't need to declare constants and you get type safety 🤔

Reply:
That sounds like a nice refactoring idea for the future Paul, not a blocker (it's merged anyway :D)